README.md

```diff
@@ -17,10 +17,10 @@ Build scripts to help generate cross ToolChains for various platforms supported
 sudo pacman -S base-devel gperf help2man bison texinfo flex gawk
 ```
 
-## For Ubuntu users:
+## For Ubuntu/Debian users:
 
 ```bash
-sudo apt-get install build-essential gperf help2man bison texinfo flex gawk
+sudo apt-get install build-essential gperf help2man bison texinfo flex gawk autoconf automake wget curl file libncurses-dev
```

Comment:
What's `file` used for btw?

Reply:
```
[INFO ]  Finalizing the toolchain's directory
[INFO ]    Stripping all toolchain executables
[22:50] / /toolchain/koxtoolchain/build/CT_NG_BUILD/lib/1.23.0.72-23ba174/scripts/build/internals.sh: line 91: file: command not found
```

Reply:
It's used by ct-ng to detect binaries/shared objects in order to figure out what to feed to strip ;).

Reply:
And, for completeness' sake: curl/wget as, surprise, download backends, and ncurses for Kconfig

(I'm not *entirely* sure if it's needed for a menu-less configure/install/sample run, though).